logging/telemetry.go

```diff
@@ -87,6 +84,8 @@ func EnsureTelemetryConfigCreated(configDir *string, genesisID string) (Telemetr
 		var err error
 		configPath, err = config.GetConfigFilePath(loggingFilename)
 		if err != nil {
+			cfg := createTelemetryConfig()
+			initializeConfig(cfg)
 			return createTelemetryConfig(), true, err
```

Comment:
I think that you meant:
`return cfg, true, err`

Reply:
Yes, thank you.